### PR TITLE
#89 Why you need to go visual v1

### PR DESCRIPTION
**Pull request by gabduss**

### CODE REVIEW

**Comment by abeggchr** [comment]
Ready for lead author review?

**Comment by gabduss** [comment]
Yes, it's now ready for lead author review.

File: articles/go-visual.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Sketch to understand
+If you work on a software development project, you will probably face complex problems on a daily basis. 
+Some common ones are: Concurrency issues, async dataflow, evolutionary architecture, communication to other applications or how do to implement a specific part of the business logic.
+Sketches can help you to understand and solve many of those issues. Let's make an example. You're a software engineer and have to implement a part of a business logic. A service that creates an overview of all your bank accounts and their credit.
+The requirement for the transformation will probably be a list of rules. Including how to handle different currencies and its exchange rate.
+To draw a workflow diagram and having it on a paper in front of you, will help you to understand the data flow and the connection between the rules. It will help you to simplify your workflow diagram. Maybe some conditions are already included in other ones and can be pruned away. Or you can simplify the workflow by changing the order in which the rules are applied.
```
Comment (peitor) [inline articles/go-visual.md]: Can you say more about drawing/sketching by hand  and modelling with a tool? What works better?
BTW: I love sketching by hand. Even important Architecture Diagrams.

Comment (gabduss) [reply articles/go-visual.md]: Hi Peter, thanks a lot for the review! I will try to improve the article with the help of your feedback.

**Pull request merged**